WIP-SQL/distinct.md

```diff
@@ -0,0 +1,121 @@
+# DISTINCT
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[More on DISTINCT](https://www.tutorialspoint.com/sql/sql-distinct-keyword.htm)'
+
+---
+## Content
+
+The `DISTINCT` keyword enables users to cut the duplicates that result after doing a `SELECT` query.
+
+Consider the following table:
+```
+EMPLOYEES table
++----+----------+-----+-----------+
+| ID | NAME     | AGE | SALARY    |
++----+----------+-----+-----------+
+|  1 | Johnny   |  32 | 2000.00   |
+|  2 | Britta   |  25 | 1500.00   |
+|  3 | Sebastian|  23 | 2000.00   |
+|  4 | Catalin  |  25 | 6500.00   |
+```
+
+Now, if we run the following query:
+```
+SELECT SALARY
+FROM EMPLOYEES
+ORDER BY SALARY;
+```
+The result will look as follows:
+```
++----------+
```

Comment:
Use the pokemon example here - you can use an example like, finding a list of pokemon types present in a filtered query, which is more typical of a use than distinct dollar amounts